src/clients/c++/perf_client/perf_client.cc

```diff
@@ -286,6 +286,7 @@ Usage(char** argv, const std::string& msg = std::string())
   std::cerr << "\t-v" << std::endl;
   std::cerr << std::endl;
   std::cerr << "I. MEASUREMENT PARAMETERS: " << std::endl;
+  std::cerr << "\t--async (-a)" << std::endl;
```

Comment:
For sequence models in sync mode... does that mean that only one request for a sequence is made at a time?

Reply:
Yes. This is because we are restricted with our blocking synchronous API. I have modified the default mode of operation to asynchronous for sequence models.